imap_processing/glows/l1/glows_l1a_data.py

```diff
@@ -157,40 +164,75 @@ class HistogramL1A:
     glows_time_offset: tuple[int, int]
         GLOWS end time offset for the block, in the form (seconds, subseconds). In
         algorithm document as "glows_end_time_offset"
+    number_of_spins_per_block: int
+        Number of spins in the block, from L0.SPINS
+    number_of_bins_per_histogram: int
+        Number of bins in the histogram, from L0.NBINS
+    number_of_events: int
+        Number of events in the block, from L0.EVENTS
+    filter_temperature_average: int
+        Average filter temperature in the block, from L0.TEMPAVG. Uint encoded.
+    filter_temperature_variance: int
+        Variance of filter temperature in the block, from L0.TEMPVAR. Uint encoded.
+    hv_voltage_average: int
+        Average HV voltage in the block, from L0.HVAVG. Uint encoded.
+    hv_voltage_variance: int
+        Variance of HV voltage in the block, from L0.HVVAR. Uint encoded.
+    spin_period_average: int
+        Average spin period in the block, from L0.SPAVG. Uint encoded.
+    spin_period_variance: int
+        Variance of spin period in the block, from L0.SPAVG. Uint encoded.
```

Comment:
similar to Greg's comment
```suggestion
        Variance of spin period in the block, from L0.SPVAR. Uint encoded.
```